file-formats/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -48,10 +48,7 @@ INTERFACE zif_aff_cota_v1
     BEGIN OF co_rfctype,
```

Comment:
I just saw that there is only one more RFC type left. Why do you keep a field with only one possible value? Couldn't we remove the field at all? I think, as long as it is not mandatory and it has a default value, it could be introduced later when the field provides multiple values.

Reply:
I do not have a strong opinion on that. Yes, field can be added compatibly since there is a default. But I'm not sure whether it should be removed or not.

Reply:
@WUEHR Do you really want to keep this in the ABAP file format? We can introduce it (IMHO in a compatible way) as soon as you provide/support different RFC types. Since you anyhow specified a default value it wouldn't be serialized into the JSON

Reply:
@schneidermic0 we have decided that we will keep the field in and hide it for the time being through the configuration include. So customer is not confused about a field which always has the same value. Is that a acceptable solution?

Reply:
Thanks for the explanation. Yes, it would be acceptable. 

However, removing the field in the future, if not needed, would be an incompatible change and would require to increase the `formatVersion`.

Just let me know how you want to go forward.